target/linux/ramips/dts/mt7628an_360-p2.dts

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "360,p2", "mediatek,mt7628an-soc";
+	model = "360 P2";
```

Comment:
So, "360" is the vendor here?

Reply:
Yes, the full name is "Qihoo 360 Technology Co. Ltd." but few of people call it "Qihoo" recent years.

Reply:
Looks like Qihoo is preferred, though: https://github.com/openwrt/openwrt/commit/a9360452f0130bb39a8e732334a174cc3f8991fd

Reply:
okay, I changed it.